src/client/containers/Layout/index.tsx

```diff
@@ -189,9 +194,12 @@ export const Layout: FC = ({ children }) => {
           walletAddress={selectedAddress}
           addressEnsName={addressEnsName}
           onWalletClick={() => dispatch(WalletActions.walletSelect({ network: currentNetwork }))}
+          disableWalletSelect={hideControls || isInIframe}
```

Comment:
Right now we also have the integration of our site as a Gnosis App that runs as an Iframe. Since it is not really considered a partner, left the iFrame checks there too.